wix-petri-core/src/test/java/com/wixpress/petri/laboratory/TransientCacheExperimentSourceTest.scala

```diff
@@ -0,0 +1,50 @@
+package com.wixpress.petri.laboratory
+
+import java.util
+
+import com.wix.hoopoe.koboshi.cache.TimestampedData
+import com.wix.hoopoe.koboshi.cache.transience.AtomicReferenceCache
+import com.wixpress.petri.ExperimentsAndState
+import com.wixpress.petri.experiments.domain.{ConductibleExperiments, Experiment, FakeClock}
+import org.joda.time.{Instant, Minutes}
+import org.specs2.matcher.Matcher
+import org.specs2.mutable.SpecificationWithJUnit
+import org.specs2.specification.Scope
+
+class TransientCacheExperimentSourceTest extends SpecificationWithJUnit {
+
+  trait Context extends Scope {
+    val underlyingCache = new AtomicReferenceCache[ConductibleExperiments]
+    val koboshiInterval = 5
+    val remoteFetchingLatencyDuration = 1
+    val aPointInTime = new Instant(5)
+    val timeToStale = Minutes.minutes(koboshiInterval).plus(remoteFetchingLatencyDuration).toStandardDuration
+  }
+
+
+  "TransientCacheExperimentSource" should {
+    "reflect data is fresh when it was updated less than 5 minutes, the koboshi fetch interval, ago" in new Context {
+      val clock = new FakeClock(aPointInTime.plus(timeToStale).minus(1).toDateTime)
+      val source = new TransientCacheExperimentSource(underlyingCache, clock)
+      underlyingCache.write(dataWithTimestamp(aPointInTime))
+
+      source.read() must beFresh()
+
+    }
+    "reflect data is stale when it was updated more than 5 minutes, the fetch interval, ago" in new Context {
+      val clock = new FakeClock(aPointInTime.plus(timeToStale).plus(1).toDateTime)
```

Comment:
the plus(1) implicitly relies on the 'remoteFetchingLatencyDuration' set above.
maybe change test name and use the 'remoteFetchingLatencyDuration' variable instead of '1' to be more clear?